plugins/item_tasks/web_client/views/PaginateTasksWidget.js

```diff
@@ -37,7 +37,7 @@ var PaginateTasksWidget = View.extend({
         if (settings.collection) {
             this.render();
         } else {
-            this.collection.fetch(this.params);
+            this.collection.fetch();
```

Comment:
Just to be clear, I see this as a bugfix. `this.params` was never documented or implemented, and should never have made it through as a property of the recently-added `PaginateTasksWidget` in the first place.